### PR TITLE
Relaxation of xorg-libx11 version requirements.

### PR DESCRIPTION
**Pull request by FelixMoelder**
As already introduced with https://github.com/conda-forge/xorg-libxext-feedstock/pull/10, this PR also allows `xorg-libx11=1.7` being required for `osx-arm64` migration.

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by FelixMoelder** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml
Comment (mbargull) [inline recipe/meta.yaml]:
```suggestion
  number: 1003
```
Bumping once per PR is fine/preferred since the intermediate commits won't issue new builds, only the final (merge) commit.

Comment (FelixMoelder) [reply recipe/meta.yaml]: Thank's for telling me. I had in mind that this would be necessary each time.

**Review by pkgw — APPROVED** [review]
Looks good to me. I'll hold off on merging until we get an OK from @mbargull.

**Review by mbargull — APPROVED** [review]
Changes LGTM!

**Pull request merged**
